### DISCUSSION TITLE
How do I change the default GPT-3.5 model to GPT-4?

### DISCUSSION THREAD

**Discussion by jackyliang** [pinecone-io/canopy#310] [Q&A]
Hello! Quick question I have here. 

I want Canopy to use GPT-4 Turbo instead of GPT-3.5 (which appears to be the default when grepping for the model name, gpt-3-turbo comes up). 

I tried to change all the configuration strings that I could find, such as in `config/default.yaml` and `src/canopy/llm/openai.py` but I still seem to be using GPT-3.5 Turbo. It also isn't clear in the documentation how to change to using GPT-4, let alone other LLMs like Cohere. 

Thank you for your help!

**Comment by mhodsaifansari** [comment]
i did it by changing gpt-3.5-turbo to gpt 4 in default.yaml, works
